src/ML/include/BipedalLocomotion/ML/MANNAutoregressive.h

```diff
@@ -27,6 +27,34 @@ namespace BipedalLocomotion
 namespace ML
 {
 
+/**
+ * MANNFootState contains the state of a foot in contact with the ground.
+ */
+struct MANNFootState
+{
+    Contacts::EstimatedContact contact; /**< Contact state */
+    Math::SchmittTriggerState schmittTriggerState; /**< Schmitt trigger state */
+    std::vector<Eigen::Vector3d> corners; /**< Corners of the foot */
+
+    /**
+     * Generate a dummy foot state.
```

Comment:
```suggestion
     * Generate a foot state.
```

Reply:
I guess this is because this function is only called when generating a dummy autoregressive state at https://github.com/ami-iit/bipedal-locomotion-framework/blob/1fc5b6d225ac997ec84e77f8082ce2848723aba5/src/ML/src/MANNAutoregressive.cpp#L450-L457 I agree that either we remove this `dummy` as suggested (along with the one in the return below) or we call the function `generateDummyFootState`. In my opinion the first is better!